src/app/app-routing.module.ts

```diff
@@ -1,7 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-const routes: Routes = [];
+import {TodoNewComponent} from "~components/todo/todo-new/todo-new.component";
```

Comment:
It will be a good practice to add spaces after/before {/}. E.x. `import { TodoNewComponent } from ...`

Reply:
Good catch :)